### DISCUSSION TITLE
Start GuidedTour at specific step index, without using stepIndex

### DISCUSSION THREAD

**Discussion by MohamedLoukili** [gilbarbara/react-joyride#1094] [Q&A] [answered]
Can I trigger the tour from a specific step (not necessary the first one) without using stepIndex ?  Since using stepIndex make you manage steps changes with callback.

**Comment by gilbarbara** [comment]
You could filter your steps to remove the ones you don't need or create a new array of steps.

**Reply by MohamedLoukili** [reply]
On my example I still need the prev button to go back on previous steps. But I was wondering if it can get be done without having to use callback to manage steps. I did it like that anyway. thank you for answering 👍

**Reply by gilbarbara** [reply, marked as answer]
Then, no. Only using the stepIndex.
